src/plugins/guided_onboarding/public/services/api.ts

```diff
@@ -282,6 +282,23 @@ export class ApiService implements GuidedOnboardingApi {
     );
   }
 
+  /**
+   * An observable with the boolean value if the step is ready_to_complete (i.e., user needs to click the "Mark done" button).
+   * Returns true, if the passed params identify the guide step that is currently ready_to_complete.
+   * Returns false otherwise.
+   * @param {GuideId} guideId the id of the guide (one of search, observability, security)
```

Comment:
Should we update this comment (probably needs to be updated in other areas of this file as well)?

Reply:
Yes, great catch @alisonelizabeth 😄  I'll include this change in the docs update PR